contrib/ubuntu-kvm

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
How about move this file into `contrib/kvm/`?  Then all files about kvm are in the same directory.

Reply:
+1. Done and resubmitted.